OloEngine/src/Platform/Vulkan/VulkanTransientUpload.cpp

```
@@ -0,0 +1,239 @@
+        barrier.dstQueueFamilyIndex = VK_QUEUE_FAMILY_IGNORED;
+        barrier.image = image;
+        barrier.subresourceRange = { VK_IMAGE_ASPECT_COLOR_BIT, baseMip, mipCount, baseLayer, layerCount };
+
+        VkDependencyInfo dep{};
+        dep.sType = VK_STRUCTURE_TYPE_DEPENDENCY_INFO;
+        dep.imageMemoryBarrierCount = 1u;
+        dep.pImageMemoryBarriers = &barrier;
+        vkCmdPipelineBarrier2(cmd, &dep);
+    }
+
+    VulkanRendererAPI* TryGetRecordingVulkanAPI()
```

Cppcheck (2.21.0)

[style] The function 'TryGetRecordingVulkanAPI' is never used.

(unusedFunction)